### PR TITLE
Added Loud Modifier

### PR DESCRIPTION
**Pull request by starryburst**
Inspired by Voyante Bavarde from Loup Garou en Ligne, a modifier which announces all of a players reports.

### CODE REVIEW

File: Games/types/Mafia/roles/cards/Loud.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                    return;
+                }
+
+                for (let alert of this.game.alertQueue) {
+                    if (!alert.recipients) {
+                        continue
+                    }
+                    for (let recipient of alert.recipients) {
+                        if (recipient == this.player) {
+                            this.game.queueAlert(`A Loud ${this.player.role.name} is overheard reading: ${alert.message}`);
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/Loud.js:24]: their role is also revealed, i think this can be added in the description

**Comment by starryburst** [comment]
Something that might not be intended is that the loud player would hear their message again, but I believe it's more efficient than looping through everyone and sending it to everyone besides them.

**Comment by pikulet** [comment]
can you make the gy participation message not be read out?

**Comment by pikulet** [comment]
alerts sent using sendAlert will not be read out, e.g. banquet messages
I think this is fine.

**Comment by pikulet** [comment]
this is good to go; just requires handling those merge conflicts

**Review by pikulet — DISMISSED** [review]

**Review by pikulet — APPROVED** [review]

**Review by pikulet — APPROVED** [review]

**Pull request merged**